src/app/home/home.component.ts

```diff
@@ -142,9 +142,13 @@ export class HomeComponent implements OnInit, DoCheck, AfterViewChecked, OnDestr
       }
       return forkJoin(obsArr);
     })).subscribe((response: any) => {
-        this.userService.unset();
-        this.router.navigate([ '/login' ], {});
-    }, err => console.log(err));
+      this.userService.unset();
+      this.router.navigate([ '/login' ], {});
+    }, catchError (error => {

```

Comment:
`catchError` should be part of the `pipe` as a second operator function passed into that part of the observable sequence